src/actions/index.js

```diff
@@ -42,45 +39,36 @@ export const OPEN_FILE_E_OFFLINE = 'OPEN_FILE_E_OFFLINE'
 export const OPEN_FILE_E_NO_APP = 'OPEN_FILE_E_NO_APP'
 export const ALERT_CLOSED = 'ALERT_CLOSED'
 
+// Temporary
+const STACK_URL = 'http://cozy.local:8080'
```

Comment:
Should we still be using this? We know the actual stack url from the inject data, and I don't think this workaround will work on the test servers?